src/registrar/tests/test_views.py

```diff
@@ -1065,6 +1069,26 @@ def test_application_edit_restore(self):
         # page = self.app.get(url)
         # self.assertNotContains(page, "VALUE")
 
+    def test_long_org_name_in_application(self):
+        """
+        Make sure the long name is displaying in the application form,
+        org step
+        """
+        request = self.app.get(reverse("application:")).follow()
+        self.assertContains(request, "Federal: an agency of the U.S. government")
```

Comment:
(optional) I think for coverage purposes it would be helpful to test on at least one or two additional long org names here.

Reply:
I feel like if one works then they all certainly work?

Reply:
It'd just be for the very rare edge case of if the dict itself gets messed with but not that value in particular. For instance, someone accidently replaces the entire dict with only the first record. Very much (optional) though, I think its stretching a bit